badge-maker/lib/badge-renderers.js

```diff
@@ -159,7 +160,10 @@ class Badge {
       }
     }
     let rightWidth = messageWidth + 2 * horizPadding
-    if (hasLogo && !hasLabel) {
+    if (logoOnly) {
+      const paddingRight = message.length ? horizPadding - 1 : 0
+      rightWidth += totalLogoWidth + paddingRight
+    } else if (hasLogo && !hasLabel) {
       rightWidth += totalLogoWidth + horizPadding - 1
```

Comment:
hi!  as `logoOnly` already contains `&& !message.length`, so one of the conditional in the `if (logoOnly)` or the `message.length ?`  is redundant.

keep the `if`

```suggestion
    if (logoOnly) {
      rightWidth += totalLogoWidth
    } else if (hasLogo && !hasLabel) {
      rightWidth += totalLogoWidth + horizPadding - 1
```

or, keep the ternary condition

```suggestion
    if (hasLogo && !hasLabel) {
      const paddingRight = message.length ? horizPadding - 1 : 0
      rightWidth += totalLogoWidth + paddingRight
```

Reply:
Thanks for pointing out!